chromadb/test/property/strategies.py

```diff
@@ -448,6 +449,29 @@ def is_valid(self, rule) -> bool:  # type: ignore
         return True
 
 
+def opposite_value(value: LiteralValue) -> SearchStrategy[Any]:
+    """
+    Strategy for generating opposite values of the given value - testing of $nin
+    """
+    if isinstance(value, float):
+        return st.floats(allow_nan=False, allow_infinity=False).filter(
+            lambda x: x != value
+        )
+    elif isinstance(value, str):
+        # allow only printable ascii characters
```

Comment:
why?

Reply:
Non-printable chars seem to be throwing causing issues with tests. I've noticed:

```python
sql_alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_"
safe_text = st.text(alphabet=sql_alphabet, min_size=1)
```

so maybe I can try using that instead.